www-servers/h2o/files/h2o.logrotate

```diff
@@ -0,0 +1,8 @@
+/var/log/h2o/*.log {
+	missingok
+	delaycompress
+	sharedscripts
+	postrotate
+		/etc/init.d/h2o reload > /dev/null 2>&1 || true
```

Comment:
You shouldn't really call this for people who aren't actually using OpenRC. I'm sorry but I don't remember off-hand how we dealt with that.